challenges/binary_search.py

```diff
@@ -0,0 +1,19 @@
+def binary_search(sorted_array, test_value):
+    """ run through sorted_array and look for test_value.
+    if test_value is in sorted_array, return index for test_value
+    in sorted_array. If test_value is not in sorted_array, return
+    -1 """
+    counter = 0
+    if len(sorted_array) = 0:
+        return -1
+    else:
+        len(sorted_array > 0 and type(test_value) == int:
```

Comment:
You have incorrect syntax here. If you are going to be using a condition, you need an `if` keyword.

Reply:
Also, you are missing the ending parenthesis on `len`, which might be why the linter is having issues.